service_discovery_manager/Component/mmBASE/BaseAPI/ifaddrs.cc

```diff
@@ -0,0 +1,221 @@
+/*
```

Comment:
All source file of this project is cpp, so please rename to ifaddrs.cpp

Reply:
done